lumispy/signals/luminescence_transientspec.py

```diff
@@ -21,21 +21,47 @@
 -------------------------------------------------
 """
 
-from hyperspy.signals import Signal2D
+from hyperspy.signals import Signal1D, Signal2D
 from hyperspy._signals.lazy import LazySignal
 
+from lumispy.signals import LumiSpectrum, LumiTransient
 from lumispy.signals.common_luminescence import CommonLumi
 from lumispy.signals.common_transient import CommonTransient
 
 
+class TransientSpectrumCasting(Signal1D, CommonLumi, CommonTransient):
+    """**1D signal class for casting reduced LumiTransientSpectrum to either Luminescence or Transient**"""
+
+    _signal_type = "TransientSpec"
+    _signal_dimension = 1
+
+    def __init__(self, *args, **kwargs):
+        if hasattr(self, "axes_manager") and self.axes_manager[-1].units in [
```

Comment:
Would it be worth using `pint` to check that the unit is of time dimension?

For example:
```python
import pint

ureg = pint.UnitRegistry()

assert ureg("ms").dimensionality == ureg("s").dimensionality

```